src/main/java/core/basesyntax/service/strategy/impl/BalanceHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.strategy.impl;
+
+import core.basesyntax.dao.impl.FruitDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.strategy.TransactionStrategy;
+
+public class BalanceHandler implements TransactionStrategy<Fruit> {
+    @Override
+    public void getOperation(Fruit fruit) {
+        FruitDaoImpl fruitDao = new FruitDaoImpl();
```

Comment:
let's initialize this field inside of the constructor, the same for other handlers